cmake/header_test.in

```diff
@@ -42,5 +42,6 @@
 // projects build with NOMINMAX this doesn't seem to be high priority to fix.
 //#define min(...) CUB_MACRO_CHECK('min', windows.h)
 //#define max(...) CUB_MACRO_CHECK('max', windows.h)
+#define small CUB_MACRO_CHECK('small', windows.h)
```

Comment:
Can you push a PR to add this to Thrust, too?

Reply:
Sure!